pkg/config/node.go

```diff
@@ -168,28 +166,21 @@ func (c *Cluster) PopulateVRIDs() error {
 		return fmt.Errorf("Cluster name can't be empty")
 	}
 	c.APIVirtualRouterID = utils.FletcherChecksum8(c.Name+"-api") + 1
-	c.DNSVirtualRouterID = utils.FletcherChecksum8(c.Name+"-dns") + 1
-	if c.DNSVirtualRouterID == c.APIVirtualRouterID {
-		c.DNSVirtualRouterID++
-	}
 	c.IngressVirtualRouterID = utils.FletcherChecksum8(c.Name+"-ingress") + 1
-	for c.IngressVirtualRouterID == c.DNSVirtualRouterID || c.IngressVirtualRouterID == c.APIVirtualRouterID {
+	for c.IngressVirtualRouterID == c.APIVirtualRouterID {
```

Comment:
Nit: This doesn't need to be a for loop anymore because there's only one other ID.